signature-aggregator/README.md

```diff
@@ -22,19 +22,21 @@ Sample config that can be used for local testing is `signature-aggregator/sample
 
 ## Interface
 
-The only exposed endpoint is `/aggregate-signatures/by-raw-message`  expecting `application/json` encoded request with the following body. Note that all the fields are optional but at least one of `data` or `justification` must be non-empty:
+The only exposed endpoint is `/aggregate-signatures`  expecting `application/json` encoded request with the following body. Note that all the fields are optional but at least one of `message` or `justification` must be non-empty:
 ```json
 {
-    "data": "",               // (string) hex-encoded unsigned message bytes to be signed
+    "message": "",            // (string) hex-encoded unsigned message bytes to be signed
     "justification": "",      // (string) hex-encoded bytes to supply to the validators as justification
  .  "signing-subnet-id": "",  // (string) hex or cb58 encoded signing subnet ID. Defaults to source blockchain's subnet from data if omitted.
-    "quorum-num": 67          // (int) quorum percentage required to sign the message. Defaults to 67 if omitted
+    "quorum-percentage": 67  // (int) quorum percentage required to sign the message. Defaults to 67 if omitted
 }
 ```
 
 ## Sample workflow
 If you want to manually test a locally running service pointed to the Fuji testnet you can do so with the following steps.
 
+Note that this might fail for older messages if there has been enough validator churn, and less then threhold weight of stake of validators have seen the message when it originated. In this case try picking a more recent message.
```

Comment:
Expanding this to 1) explain that the Warp protocol does not support resigning, but 2) protocols on top of Warp such as Teleporter do (and link to the [relevant docs](https://github.com/ava-labs/teleporter/pull/487/files#diff-633988a43ea67f810d01288f07e014281067fe8129b594e93e550fa49f07e580R77))

Reply:
I just left a comment on your PR but I will link to the doc section once they are merged in main.  I don't really want to over-elaborate here beyond linking since this is just a sample testing flow to make sure that the service works as expected.